task10_5/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task10_5
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите натуральное число m:");
+            int m;
+
+            if (!TryInputNumber(out m))
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            Console.WriteLine("Введите натуральное число n:");
+            int n;
+
+            if (!TryInputNumber(out n))
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            int min;
+            if (n<=m) min = n;    
+            else min = m;
+            Console.WriteLine("Все взаимно простые числа с m и n:");
+
+            for (int i = 1; i <= min; i++)
```

Comment:
Вы невнимательно прочитали условие.

> Написать программу, которая определяет все натуральные числа, не превосходящие _mn_ ...

```suggestion
            for (int i = 1; i <= m * n; i++)
```